libcudacxx/include/cuda/std/__cccl/builtin.h

```diff
@@ -414,6 +414,19 @@
 #  define _CCCL_BUILTIN_FMINL(...) __builtin_fminl(__VA_ARGS__)
 #endif // _CCCL_CHECK_BUILTIN(builtin_fmin)
 
+#if _CCCL_CHECK_BUILTIN(builtin_fmod) || _CCCL_COMPILER(GCC)
+#  define _CCCL_BUILTIN_FMODF(...) __builtin_fmodf(__VA_ARGS__)
+#  define _CCCL_BUILTIN_FMOD(...)  __builtin_fmod(__VA_ARGS__)
+#  define _CCCL_BUILTIN_FMODL(...) __builtin_fmodl(__VA_ARGS__)
+#endif // _CCCL_CHECK_BUILTIN(builtin_fmod)
+
+// clang-cuda fails with fatal error: error in backend: Undefined external symbol "modf"
```

Comment:
Suggestion: add which version of clang failed, so we can revisit this in the future.

Reply:
This is failing for all of them. I would like to add some when we actually have one passing